src/components/MovieCard/MovieCard.tsx

```diff
@@ -1,7 +1,38 @@
 import React from 'react';
 
 import './MovieCard.scss';
+import Movie from '../../types/Movie';
 
-export const MovieCard: React.FC = () => (
-  <>Put the card here</>
+interface Props {
+  card: Movie;
+}
+
+const MovieCard: React.FC<Props> = ({ card }) => (
```

Comment:
Use named imports/exports everywhere to fix these problems
![image](https://user-images.githubusercontent.com/96409563/225838499-33a6775f-cadf-42c6-bdd8-fb2047723602.png)